### PR TITLE
TWIRF7

### PR DESCRIPTION
**Pull request by dns2utf8**
- [x] First draft
- [x] Sponsor text
- [x] Outline ticket sale
- [ ] Maybe: some thoughts about this run at diversity tickets
- [x] Hint impl days
- [x] Mention hotel quota Züri Tourismus

### CODE REVIEW

**Comment by gandro** [comment]
Another potential topic: Mention the hotel quota from Zurich Tourism again. They kindly asked me to promote it a bit, since registrations there are pretty slow.

File: _posts/2017-08-17-this-week-in-rustfest-7-main-sponsor-parity.md [hunk 1]
```diff
@@ -22,9 +22,22 @@ TODO: outline timeline for sales
...
+Roughly speaking it means that around every rust conference this fall some people may work on rust and central crates near the venue for a day or two.
+These "impl days" would be interesting because of the concentrated knowledge.
+
+Imagine: You are working on a problem and use a crate.
+What if you could ask the author of said crate if the crate fits the problem or how to extend it.
+
+All of this is still very vague but: If you consider to join, maybe add a day or two after RustFest Zürich.
+Worst case you will get two days of vacation time in switzerland.
+
+![Image featuring a mountain and a house in the alps](http://www.fotos.sc/img2/u/ohara62/h/Berge__Wolken__Sonne_.jpg)
```
Comment (Hoverbear) [inline _posts/2017-08-17-this-week-in-rustfest-7-main-sponsor-parity.md]: We shouldn't hotlink images or use anything we don't have the rights to.

File: _posts/2017-08-17-this-week-in-rustfest-7-main-sponsor-parity.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "This week in RustFest 7: Main sponsor Parity.io"
+authors:
+  - dns2utf8
+---
+
+RustFest is a lot of hard work and community craftsmanship [citation needed](#).
```
Comment (Hoverbear) [inline _posts/2017-08-17-this-week-in-rustfest-7-main-sponsor-parity.md]: Let's try to avoid self-back-patting.

**Comment by Hoverbear** [comment]
Rendered: screen shot 2017-08-17 at 20 26 20-fullpage

**Review by Hoverbear — APPROVED** [review]

**Comment by gandro** [comment]
It's happening! :) Looks good to me!

**Pull request merged**